src/nvim/syntax.c

```diff
@@ -6470,8 +6470,7 @@ const char *const highlight_init_cmdline[] = {
   "default link NvimInvalidOptionSigil NvimInvalidIdentifier",
   "default link NvimInvalidOptionName NvimInvalidIdentifier",
   "default link NvimInvalidOptionScope NvimInvalidIdentifierScope",
-  "default link NvimInvalidOptionScopeDelimiter "
-  "NvimInvalidIdentifierScopeDelimiter",
+  "default link NvimInvalidOptionScopeDelimiter NvimInvalidIdentifierScopeDelimiter",
```

Comment:
@justinmk, is there any chance this could be slightly related to https://github.com/neovim/neovim/issues/10241?

Reply:
no, but thanks for checking :) This is a good change to make. The line-length is not important enough to justify this quirky formatting.